rfc/rfc-61/rfc-61.md

```diff
@@ -0,0 +1,240 @@
+```sql
+call create_snapshot_view(table => 'hudi_table', commit_Time => 'commit_timestamp_from_timeline', snapshot_table => 'snapshot_hive_table');
+```
+
+| Parameter Name   | Required | Default                       | Remarks                                                                                       |
+|------------------| -------  |-------------------------------|-----------------------------------------------------------------------------------------------|
+| `table`          | `true`   | `--`                          | the Hive table name you want to create savepoint, must be a Hudi table, without database name |
+| `commit_Time`    | `false`  | `None`                        | the commit timestamp from Hudi timeline, if not provided, will use the latest commit          |
+| `user`           | `false`  | `""`                          | the user name will be saved in Hudi savepoint metadata                                        |
+| `comments`       | `false`  | `""`                          | the comment will be saved in Hudi savepoint metadata                                          |
+| `snapshot_table` | `false`  | `$table name + _$commit_time` | the snapshot view table name in hive                                                          |
+| `hms`            | `false`  | `None`                        | Hive metastore server used for syncing savepoint information                                  |
+
+* Delete Snapshot View
+  call delete savepoint command(via spark-sql or hudi-cli), meanwhile delete associated Hive table
+```sql
+call delete_snapshot_view(table => 'hudi_table', snapshot_table => 'snapshot_hive_table');
+```
+
+|  Parameter Name  | Required | Default | Remarks |
+|  --------------  | -------  | ------- | ------- |
+| `table`          | `true`   | `--`    | the Hive table name you want to delete savepoint, must be a Hudi table |
+| `instant_time`   | `true`   | `--`    | the savepoint timestamp from Hudi timeline, must provide |
+| `hive_sync`      | `false`  | `false` | whether to delete savepoint timestamp from Hive table serde properties |
+| `hms`            | `false`  | `None`  | Hive metastore server used for deleting savepoint information |
+
+* List Snapshot View
+```sql
+call show_snapshotviews(table => 'hudi_table');
+```
+
+|  Parameter Name  | Required | Default | Remarks |
+|  --------------  | -------  | ------- | ------- |
+| `table`          | `true`   | `--`    | the Hive table name you want to show savepoint list, must be a Hudi table |
+
+### Mor support
+Savepoint is already support Merge-On-Read table
+
+### Precise Event time Snapshot On Merge-On-Read table
+
+handle drifted data issue 
+
+## Rollout/Adoption Plan
+there should be no impact on existing users
```

Comment:
This section should clarify how the feature is to be rolled out. What will happen if users enable or disable it